x/recovery/keeper/params.go

```diff
@@ -8,7 +8,7 @@ import (
 
 // GetParams returns the total set of recovery parameters.
 func (k Keeper) GetParams(ctx sdk.Context) (params types.Params) {
-	k.paramstore.GetParamSetIfExists(ctx, &params)
+	k.paramstore.GetParamSet(ctx, &params)
```

Comment:
revert change

Reply:
@fedekunze 
it seems that this function GetParamSetIfExists() is sdk 0.45.7 only. Would you like me to write a custome one which mimics GetParamSetIfExists()?

Reply:
the newest v0.46.1 release has it btw